packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -485,20 +486,18 @@ export default class NewVm extends BaseComponent {
       }
     })
 
-    const VIFs = []
+    let VIFs = []
     forEach(template.VIFs, vifId => {
       const vif = getObject(storeState, vifId, resourceSet)
       VIFs.push({
         network:
           pool || isInResourceSet(vif.$network)
             ? vif.$network
-            : this._getDefaultNetworkId(template),
+            : this._getDefaultNetworkIds(template)[0],
```

Comment:
Call this only once before the loop. (and use the result [there](https://github.com/vatesfr/xen-orchestra/pull/3958/files#diff-a255ba66f114f6e11cd0862a0543b7dfR500) as well)